docs/docs/configuration/hardware_acceleration.md

```diff
@@ -65,24 +65,31 @@ Or map in all the `/dev/video*` devices.
 
 ## Intel-based CPUs
 
-### Via VAAPI
+**Recommended hwaccel Preset**
 
-VAAPI supports automatic profile selection so it will work automatically with both H.264 and H.265 streams. VAAPI is recommended for all generations of Intel-based CPUs.
-
-```yaml
-ffmpeg:
-  hwaccel_args: preset-vaapi
-```
+| CPU Generation | Intel Driver | Recommended Preset | Notes                               |
```

Comment:
Might be good to add a link to Intel's site so users can figure out what generation cpu they have: https://www.intel.com/content/www/us/en/support/articles/000005505/processors.html